server/athenian/api/controllers/settings.py

```diff
@@ -91,6 +91,48 @@ def _cached_release_settings_sql(account: int, repos: Collection[str]) -> Select
         return select([ReleaseSetting]).where(and_(ReleaseSetting.account_id == account,
                                                    ReleaseSetting.repository.in_(repos)))
 
+    @staticmethod
+    @lru_cache(1024)
+    def _cached_release_settings_for_repos_sql(repos: Collection[str]) -> Select:
+        return select([ReleaseSetting]).where(ReleaseSetting.repository.in_(repos))
+
+    @classmethod
+    async def _list_release_matches_for_repos(
```

Comment:
Therefore I believe that it should be possible to refactor the experiment to leave this class intact.